lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -28,6 +28,11 @@
 
 #include <libopencm3/stm32/rtc.h>
 
+static uint8_t _rtc_dec_to_bcd(uint8_t dec)
```

Comment:
I didn't want to make this a `#define` unless this is desirable here. Additionally, this is not intended to be a user-facing function unless people believe users will want to use this.